### DISCUSSION TITLE
Cannot create new profile

### DISCUSSION THREAD

**Discussion by resnikov** [magicbug/Cloudlog#929] [Q&A] [answered]
Have done a fresh install on NGNIX on a raspberry pi. Logged in as the m0abc user, created a new user, loged in as them, deleted the m0abc user, done the country files update and went to create a new station profile but the is no option to create one.
![image](https://user-images.githubusercontent.com/34718437/110504861-7e587000-80f5-11eb-8042-e6eae8c0e753.png)

**Comment by magicbug** [comment, marked as answer]
I made some UI changes to Station Profile this afternoon and placed the create station profile button inside an if without noticing, this has now been resolved in the code base if you grab it again

**Reply by resnikov** [reply]
Cheers, that worked :D
